charts/gardener-extension-csi-driver-lvm/templates/poddisruptionbudget.yaml

```
@@ -0,0 +1,14 @@
+{{- if gt (int .Values.replicaCount) 1 }}
+apiVersion: policy/v1beta1
```

Not served anymore since K8s 1.25. If you want this in another project of us, you can also correct it. :sweat: 

```suggestion
apiVersion: policy/v1
```